### PR TITLE
HOCS-3087 Use new workflow healthcheck in docker compose

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

**Review by philbarton — APPROVED** [review]
Looks good to me

**Review by philbarton — APPROVED** [review]
Looks good to me

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -421,6 +421,11 @@ services:
       HOCS_SEARCH_SERVICE: "http://documents:8080"
 
   case_creator:
+    healthcheck:
```
Comment (philbarton) [inline docker/docker-compose.yml:424]: This health check isn't needed.

**Review by philbarton — CHANGES_REQUESTED** [review]

File: docker/docker-compose.yml (hunk 1 above)
Comment (gardnersam) [reply docker/docker-compose.yml]: I've removed it

**Review by philbarton — APPROVED** [review]
lgtm

**Pull request merged**